<?xml version="1.0"?>
<launch>

    <!-- NOTE: THe depthmap topic musth be /depthmap -->
    <!-- But, Sachin's rosbag is /depth_map. Please change the topic name when it gets fixed-->
    <arg name="in_depthmap_topic" default="/depth_map"/>
    <arg name="out_cloud_topic" default="/cloud_test"/> 
    <arg name="subsample_factor" default="10"/>

    <node pkg="graph_slam" name="depth_to_pc_test_node" type="depth_to_pc_test_node" output="screen">
        <rosparam file="$(find graph_slam)/config/intrinsics.yaml"/>
        <rosparam file="$(find graph_slam)/config/pointcloud_filters.yaml"/>
        <param name="in_depthmap_topic" value="$(arg in_depthmap_topic)"/>
        <param name="out_cloud_topic" value="$(arg out_cloud_topic)"/>
        <param name="subsample_factor" value="$(arg subsample_factor)"/>
    </node>

</launch>
